src/tests/contract/routes/AuthorizationRoute.ts

```diff
@@ -0,0 +1,16 @@
+import express from "express";
+import asyncify from "express-asyncify";
+import { lambdaHandler } from "../../../AuthorizationCodeHandler";
+
+process.env.SESSION_TABLE = "session-table";
+process.env.TXMA_QUEUE_URL = "txma-queue";
+process.env.ISSUER = "issuer";
+process.env.USE_MOCKED = "true";
+
+export const authRouter = asyncify(express.Router());
+
+// eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
This is another thing that I just did for the POC - please can you look into how we can remove it?